massa-node/base_config/openrpc.json

```diff
@@ -1988,6 +1989,10 @@
                     "max_gas": {
                         "description": "Maximum amount of gas that the execution of the contract is allowed to cost.",
                         "type": "number"
+                    },
+                    "datastore": {
+                        "description": "A tuple which contains (key, value)",
+                        "$ref": "#/components/schemas/DataStore"
```

Comment:
This object doesn't have the same format as described in https://github.com/massalabs/massa/pull/3450#issuecomment-1400007706

Reply:
Done

Reply:
Done